aptos-move/block-executor/src/executor.rs

```diff
@@ -790,13 +790,6 @@ where
             }
         }
 
-        if ret.len() == num_txns {
```

Comment:
is this related?

Reply:
Good catch, this is actually remnant of previous change I was trying. Let me add it back.